ui/cap-react/src/antd/forms/templates/ObjectFieldTemplate.js

```diff
@@ -83,65 +79,52 @@ const ObjectFieldTemplate = ({
   return (
     <fieldset style={{ margin: "0 12px" }} id={idSchema.$id}>
       <Row gutter={rowGutter}>
-        {uiSchema["ui:title"] !== false &&
-          (uiSchema["ui:title"] || title) && (
-            <Col
-              style={{
-                padding: "0",
-                marginBottom: "12px",
-              }}
-              className={labelColClassName}
-              span={24}
-            >
-              {
-                <Divider
-                  id={`${idSchema.$id}-title`}
-                  orientation="left"
-                  style={{ margin: 0 }}
-                >
-                  <Text strong>{uiSchema["ui:title"] || title}</Text>
-                </Divider>
-              }
-            </Col>
-          )}
-        {uiSchema["ui:description"] !== false &&
-          (uiSchema["ui:description"] || description) && (
-            <Col span={24} style={DESCRIPTION_COL_STYLE}>
-              <DescriptionField
-                description={uiSchema["ui:description"] || description}
-                id={`${idSchema.$id}-description`}
-              />
-            </Col>
-          )}
+        <Col
+          style={{
+            padding: "0",
+            marginBottom: "12px",
+          }}
+          className={labelColClassName}
+          span={24}
+        >
+          <FieldHeader
+            label={uiSchema["ui:title"] || title}
```

Comment:
let's also add this `uiSchema["ui:title"] !== false` so we can keep the title hiding feature